<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ƒê·ªïi m·∫≠t kh·∫©u</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #2c2c2c;
        color: white;
        font-size: 14px;
    }
    QLineEdit {
        background-color: #3c3c3c;
        border: 1px solid #555;
        padding: 6px;
        border-radius: 6px;
        color: white;
    }
    QPushButton {
        background-color: #4a90e2;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #357ABD;
    }
   </string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;üîí ƒê·ªïi m·∫≠t kh·∫©u t√†i kho·∫£n&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="oldPasswordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>M·∫≠t kh·∫©u hi·ªán t·∫°i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="eyeButtonOld">
          <property name="text">
           <string>üëÅÔ∏è</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="newPasswordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>M·∫≠t kh·∫©u m·ªõi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="eyeButtonNew">
          <property name="text">
           <string>üëÅÔ∏è</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="confirmPasswordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Nh·∫≠p l·∫°i m·∫≠t kh·∫©u m·ªõi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="eyeButtonConfirm">
          <property name="text">
           <string>üëÅÔ∏è</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="changePasswordButton">
        <property name="text">
         <string>ƒê·ªïi m·∫≠t kh·∫©u</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>H·ªßy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
